### DISCUSSION TITLE
HwV380E22_WF3_MINI_Q7_20201023

### DISCUSSION THREAD

**Discussion by pingflood** [Arkady23/V380-Web#43] [General]
Hello my friend,

I'm here to report another camera data.

I have tested with this V380 Door Eye model: https://aliexpress.com/item/4000429796084.html

V380 App identify it as:
AppV380E22_V3.3.2.9#YW01_20210112
KerV380E22_MINI_Q7_V3.1.1_20181220
HwV380E22_WF3_MINI_Q7_20201023

Wireshark data:
{"soft_id":"V380E3_C5#YW","dev_id":,"version":303020901,"timestamp":1654297119}

It **won't work** with current
V380 Door Eye | HwV380E22_WF3_MINI_Q7 | V380E3_C5#YW | httpd_V380E3_C5.YW.zip

Hope this info can help your amazing project.

Kind regards

**Comment by Arkady23** [comment]
Try this variant1. Write did it work out!
[test1.zip](https://github.com/Arkady23/V380-Web/files/8839447/test1.zip)

If the web server does not work, you can try this patch: 
[test2.zip](https://github.com/Arkady23/V380-Web/files/8839461/test2.zip)

**Reply by pingflood** [reply]
Just tested both but unfortunately they don't work yet. It simply won't run on the camera.

I dismantled it looking for UART but there's no exposed pins for that. :(

I've also asked v380technical for an update, but no answer so far.

Anyway, thanks for trying to provide an update

**Comment by Arkady23** [comment]
Added a description for installing a web server manually for new cameras.

**Reply by pingflood** [reply]
Hey my friend!

Just passing by to say that I finally got some time to test it and the manual install is working perfectly now.

Keep up the amazing work :clap: 

Kind regards
